tests/unit/data_formats/arrow/test_arrow_format.py

```diff
@@ -343,6 +343,51 @@ def test_can_detect(self, fxt_test_datumaro_format_dataset, test_dir):
         detected_formats = Environment().detect_dataset(test_dir)
         assert [self.importer.NAME] == detected_formats
 
+    @pytest.mark.parametrize(
+        ["fxt_dataset", "save_media"],
+        [
+            pytest.param(
+                "fxt_image",
+                True,
+                id="image_with_media",
+            ),
+            pytest.param(
+                "fxt_point_cloud",
+                True,
+                id="point_cloud_with_media",
+            ),
+            pytest.param(
+                "fxt_image",
+                False,
+                id="image_without_media",
+            ),
+            pytest.param(
+                "fxt_point_cloud",
+                False,
+                id="point_cloud_without_media",
+            ),
+        ],
+    )
+    def test_media_contents(self, fxt_dataset, save_media, test_dir, helper_tc, request):
+        fxt_dataset = request.getfixturevalue(fxt_dataset)
+
+        fxt_dataset.export(test_dir, format=self.format, save_media=save_media)
+        imported_dataset = Dataset.import_from(test_dir)
+        for item_a, item_b in zip(fxt_dataset, imported_dataset):
+            if isinstance(item_a.media, FromFileMixin):
+                helper_tc.assertTrue(item_a.media.bytes is not None)
+            helper_tc.assertTrue(item_a.media.data is not None)
+            if save_media:
+                helper_tc.assertTrue(item_b.media.bytes is not None)
+                helper_tc.assertTrue(item_b.media.data is not None)
+            else:
+                if isinstance(item_a.media, FromFileMixin):
+                    helper_tc.assertTrue(item_b.media.bytes is not None)
+                    helper_tc.assertTrue(item_b.media.data is not None)
+                else:
+                    helper_tc.assertTrue(item_b.media.bytes is None)
+                    helper_tc.assertTrue(item_b.media.data is None)
```

Comment:
`helper_tc` is made for bridging new `pytest` tests with the existing test code based on `unittest`, e.g. `compare_datasets()`. In this test, you can use `assert a == b` as `pytest`'s standard.

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/986/commits/64aa4b28ff20d0b13d2b1e02c95367b904740bd3